### PR TITLE
Added cards for Courses and People

### PR DESCRIPTION
**Pull request by OmarHHammad**

### CODE REVIEW

**Review by TerryvilleDev — COMMENTED** [review]
Could you explain why you are commented out the majority of my post method for the CreateUser view?

**Comment by OmarHHammad** [comment]
> Could you explain why you are commented out the majority of my post method for the CreateUser view?

I uncommented your code, but it seems that your code throws errors. I had it commented out so that I could still work. Here's the error:

Screen Shot 2021-12-16 at 3 33 47 PM

**Comment by TerryvilleDev** [comment]
> > Could you explain why you are commented out the majority of my post method for the CreateUser view?
> 
> I uncommented your code, but it seems that the CreateUser throws errors. I had it commented out so that I could still work. Here's the error: Screen Shot 2021-12-16 at 3 37 43 PM

That's weird, I'm running the same code of my machine and it runs just fine.  I'll look into why that's the case when I get a chance.

**Comment by OmarHHammad** [comment]
It might be an issue in my end too. We can take a look at it together at our next meeting

**Review by bptome — APPROVED** [review]

File: TAInformation/views.py [hunk 1]
```diff
@@ -125,7 +122,7 @@ def get(self, request):
     # Postcondition: Creates new user, if data entered validates successfully and user doesn’t already exist.
     # Side Effects: Message indicating result is displayed at the bottom of the “Create Courses” form
     def post(self, request):
-        # Extract data from form
+        Extract data from form
```
Comment (TerryvilleDev) [inline TAInformation/views.py:125]: This is a comment for the block of code to follow.  It should be commented out.

Comment (OmarHHammad) [reply TAInformation/views.py]: Fixed

**Comment by TerryvilleDev** [comment]
> It might be an issue in my end too. We can take a look at it together at our next meeting
> […](#)
> On Dec 16, 2021, at 3:57 PM, Terence Lee ***@***.***> wrote: ﻿ Could you explain why you are commented out the majority of my post method for the CreateUser view? I uncommented your code, but it seems that the CreateUser throws errors. I had it commented out so that I could still work. Here's the error: That's weird, I'm running the same code of my machine and it runs just fine. I'll look into why that's the case when I get a chance. 

I posted this in the general channel on Discord, but you have been getting this error because you're using an earlier version of Python.  Match statements were added in the newest version (version 3.10).

**Pull request merged**